### PR TITLE
build: bump agent-js v1.3.0 and remove retryTimes default 10

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Bump agent-js v1.3.0 and remove `retryTimes` default value to 10 given that the issue is fixed in agent-js.

### CODE REVIEW

File: packages/utils/README.md
Comment (lmuntaner) [inline packages/utils/README.md:1]: What happened with these changes?

**Review by lmuntaner — APPROVED** [review]
LGTM! A minor question regarding a diff not related to the PR.

File: packages/utils/README.md
Comment (peterpeterparker) [reply packages/utils/README.md:1]: You mean why the order is changing? Good question, don't really know. I added `nonNullish` to `createAgent` so I'm guessing the compilation tree changed and therefore the order changed.

I opened an issue in the documentation library: https://github.com/peterpeterparker/tsdoc-markdown/issues/30

Comment (lmuntaner) [reply packages/utils/README.md:1]: Thanks for the explanation!

**Pull request merged**